component/card/card.css

```diff
@@ -74,14 +76,12 @@
 .card h2 {
   margin: 0.6rem 0 0.6rem 0;
```

Comment:
for these types of value you should use pixel.

Reply:
noted will work on this